### PR TITLE
Insert into empty readme files

### PR DESCRIPTION
**Pull request by finnp**
It would be nice if the readme-badger would allow empty files as well. All this needs is the assertion to be modified.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -92,4 +92,11 @@ describe('readme-badger', function() {
 
     assert.equal(result, after);
   });
+
+  it('inserts into empty files', function () {
+    var before = ''
+    var after = '\n\n[![Join the chat at https://gitter.im/myorg/myrepo](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/myorg/myrepo?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)'
```
Comment (MadLittleMods) [inline test/index.js]: Let's put the before/after in a separate fixture file like the other tests

**Comment by MadLittleMods** [comment]
@finnp Thanks for the contribution! ❤️

**Comment by finnp** [comment]
Thank you for being to responsive! 👏

**Comment by finnp** [comment]
@MadLittleMods Could you also release this change to npm? 😊

**Comment by MadLittleMods** [comment]
@finnp I am not on the contributor list for this project on npm yet.

cc @trevorah https://www.npmjs.com/package/readme-badger

**Comment by finnp** [comment]
@MadLittleMods @trevorah Little nudge here ☺️ Would be nice here to get the new version out.

**Pull request merged**